src/wazuh_db/wdb.h

```diff
@@ -630,7 +630,7 @@ int wdb_reset_agents_connection(int *sock);
  * @param[in] sock The Wazuh DB socket connection. If NULL, a new connection will be created and closed locally.
  * @return Pointer to the array, on success. NULL on errors.
  */
-int* wdb_get_agents_by_connection_status(const char* status, int *sock);
+int* wdb_get_agents_by_connection_status(const char* connection_status, int *sock);
```

Comment:
Ditto, add a short brief of the chunks behavior.

Reply:
Updated!